azext_edge/edge/providers/orchestration/resources/dataflows.py

```diff
@@ -21,16 +24,118 @@
 
+    def create(
+        self,
+        name: str, 
+        instance_name: str, 
+        resource_group_name: str, 
+        profile_instances: Optional[int] = 1, 
+        log_level: Optional[str] = 'info', 
+        **kwargs,
+    ):
+
+        resource ={
+            "properties": {
+                "diagnostics": {
+                    "logs": {
+                        "level": log_level
+                    },
+                },
+                "instanceCount": profile_instances,
+            },
+            "extendedLocation": self.instances.get_ext_loc(
+                name=instance_name, resource_group_name=resource_group_name
+            )
+        }
+        
+        with console.status(f"Creating {name}..."):
+            poller = self.ops.begin_create_or_update(
+                instance_name=instance_name,
+                dataflow_profile_name=name,
+                resource_group_name=resource_group_name,
+                resource=resource
+            )
+            return wait_for_terminal_state(poller, **kwargs)
+        
+    def update(
+            self,
+            name: str,
+            instance_name: str,
+            resource_group_name: str,
+            profile_instances: Optional[int] = None,
+            log_level: Optional[str] = None,
+            **kwargs,
+    ):
+        # get the existing dataflow profile
+        original_profile = self.show(
+            name=name,
+            instance_name=instance_name,
+            resource_group_name=resource_group_name
+        )
+
+        # update the properties
+        if profile_instances:
+            original_profile["properties"]["instanceCount"] = profile_instances
+        if log_level:
+            original_profile["properties"]["diagnostics"]["logs"]["level"] = log_level
+
+        with console.status(f"Updating {name}..."):
+            poller = self.ops.begin_create_or_update(
+                instance_name=instance_name,
+                dataflow_profile_name=name,
+                resource_group_name=resource_group_name,
+                resource=original_profile
+            )
+            return wait_for_terminal_state(poller, **kwargs)
+
+    def delete(
+        self,
+        name: str,
+        instance_name: str,
+        resource_group_name: str,
+        confirm_yes: Optional[bool] = None,
+        **kwargs
+    ) -> dict:
+        dataflows = self.dataflows.list(
+            dataflow_profile_name=name,
+            instance_name=instance_name,
+            resource_group_name=resource_group_name
+        )
+        dataflows = list(dataflows)
+
+        if (name == "default"):
+            logger.warning("Deleting the 'default' dataflow profile may cause disruptions.")
+
+        if len(dataflows) > 0:
+            console.print("Deleting this dataflow profile will also delete the associated dataflows:")
+            for dataflow in dataflows:
+                console.print(f"\t{dataflow['name']}")
```

Comment:
See how this looks when the name is prefixed with a dash, dot or bullet.

Reply:
dash works. makes it easier to quickly see separate list entries